bedrock/firefox/redirects.py

```diff
@@ -128,16 +128,12 @@ def mobile_app(request, *args, **kwargs):
     redirect(r"^firefoxos", "/firefox/os/"),
     # bug 1438302
     no_redirect(r"^firefox/download/thanks/?$"),
-    # Bug 1006616
-    redirect(r"^download/?$", "firefox.new"),
-    # Bug 1409554
-    redirect(r"^(firefox|mobile)/download", "firefox.new"),
     # bug 837883
     redirect(r"^firefox/firefox\.exe$", "mozorg.home", re_flags="i"),
     # bug 821006
     redirect(r"^firefox/all(\.html)?$", "firefox.all"),
     # bug 727561
-    redirect(r"^firefox/search(?:\.html)?$", "firefox.new"),
+    redirect(r"^firefox/search(?:\.html)?$", "firefox.download"),
```

Comment:
We can probably remove nearly all of these redirects, but I thought it best to leave that to a separate PR as there are a couple we might want to keep (such as /firefox/channel/)